src/views/registration-js/attendees/fields/birth.php

```diff
@@ -6,13 +6,17 @@
  * Override this template in your own theme by creating a file at:
  * [your-theme]/tribe/tickets/registration-js/attendees/fields/birth.php
  *
- * @since   4.12.1
+ * @version 4.12.1
```

Comment:
```suggestion
 * @version TBD
```

Nice! One small thing, When we work on templates we should set the version as TBD, so when we pack things for the release with https://github.com/moderntribe/tribe-product-utils we can set the correct version.

Reply:
Ah! So:

- `@since` is used to inform when that template was introduced.
- `@version` is used to inform what changed between each version

Is that accurate?

Reply:
So:

-  `@since` is to state in which version the template was introduced. And then to list the changes introduced `@since` version.
- `@version` is the version of the template. If we make changes on a template, it means that the version should be updated to the release version. By leaving it as TBD, when we make the package and release we ensure that's being set like it should.

This info is used, for example, in "Events > Help" to let users know if we introduced changes to templates or something. And ideally if they're using their own templates, we could warn them when they are out of sync with our own versions.

To set an example, let's say a template was introduced in `1.2.3` and we're adding changes to... add bold to a title. So, the changes would be:

```
@since 1.2.3
@since TBD We are adding bold to a title.

@version TBD
```

Then when we do the release tasks, the TBD will be replaced with the number we're using for the release.

Reply:
Awesome, thanks for the explanation! :100: